src/controllers/signAccountOp/signAccountOp.ts

```diff
@@ -205,6 +201,13 @@ export class SignAccountOpController extends EventEmitter {
     this.rbfAccountOps = {}
     this.signedAccountOp = null
     this.replacementFeeLow = false
+    this.bundlerSwitcher = new BundlerSwitcher(
+      network,
+      () => {
+        return this.status ? this.status.type : null
+      },
```

Comment:
Are you sure that `this.status` is being updated? Also, can't we do it like the portfolio's `getLatestPortfolioState`